src/bin/upgrade/main.rs

```diff
@@ -158,11 +172,10 @@ fn main() {
     };
 
     if let Err(err) = output {
-        writeln!(
-            io::stderr(),
-            "Command failed due to unhandled error: {}\n",
-            err
-        ).unwrap();
+        let mut stderr = io::stderr();
+
+        writeln!(stderr, "Command failed due to unhandled error: {}\n", err).unwrap();
```

Comment:
I guess error chain isn't displayed here, we can call [display_chain](https://github.com/rust-lang-nursery/error-chain/blob/master/examples/quickstart.rs#L59) here or use [quick_main](https://docs.rs/error-chain/0.11.0-rc.2/error_chain/macro.quick_main.html).

Reply:
Thanks - I was getting confused. I think we still want to format the error message somewhat, so I've not used `display_chain()` or `quick_main!`